forc-pkg/src/pkg.rs

```diff
@@ -561,11 +589,21 @@ fn validate_dep_manifest(
     dep: &Pinned,
     dep_manifest: &PackageManifestFile,
     dep_edge: &Edge,
+    workspace_build: bool,
```

Comment:
Perhaps rather than piping this `bool` all the way through validation, we can create a new `DepKind` variant that represents the workspace member connection, and use these for the edges from the workspace node to the members? E.g. `DepKind::WorkspaceMember`?

Reply:
Ohh yeah, it is much much better thanks 👌